servo-build-dependencies/init.sls

```diff
@@ -59,19 +59,6 @@ servo-darwin-install-autoconf213-and-fix-links:
     - require:
       - pkg: servo-dependencies
 {% else %}
-multiverse:
```

Comment:
I've realized as part of #577 that we'll need to keep this code here that enables `multiverse`. Back out the change removing it, and let's see if Travis likes it better! Hopefully we won't need to change `ubuntu/files/sources.list`.